test/test-call-forwarding

```diff
@@ -171,4 +196,8 @@ def main(args):
 
 if __name__ == "__main__":
 	args = parse_arguments()
-	main(args)
+
+	try:
+		main(args)
+	except KeyboardInterrupt as e:
```

Comment:
I think making sure it exits would remove the feeling that something went wrong when executing the script. An alternative could be to print something like "Waiting for property changes, press Ctrl+C when finished" if you prefer that.